pdata/internal/cmd/pdatagen/internal/pcommon_package.go

```diff
@@ -207,6 +208,16 @@ var int64Slice = &primitiveSliceStruct{
 	testNewVal:           "1, 5, 3",
 }
 
+var intSlice = &primitiveSliceStruct{
```

Comment:
This is needed because if we rely on an `uint64`, we have a gosec issue due to typing it into an `int`.